README.md

```diff
@@ -5,3 +5,16 @@ A sidecar container that
 
 1. Registers the containerized CSI driver with kubelet (in the future).
 2. Adds the drivers custom `NodeId` (retrieved via `GetNodeID` call) to an annotation on the Kubernetes Node API Object.
+
+## Community, discussion, contribution, and support
+
+Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+
+You can reach the maintainers of this project at:
```

Comment:
maybe link the CSI Working group slack channel as well

Reply:
> maybe link the CSI Working group slack channel as well

Added. However, I cannot find the CSI working group here: https://github.com/kubernetes/community/blob/master/sig-list.md#master-working-group-list.

Is this officially a "working group"?

Reply:
Also, is there a mailing list for this wg specifically?

Reply:
https://groups.google.com/forum/#!forum/kubernetes-sig-storage-wg-csi